scrapers/stackoverflow.js

```diff
@@ -0,0 +1,70 @@
+var unirest = require("unirest");
+
+if (process.argv.length <= 2) {
+  console.log("Provide question as argument");
+  process.exit(-1);
+}
+
+const question = process.argv[2];
+
+const SEARCH_URL = "https://api.stackexchange.com/2.2/search/advanced?order=desc&sort=activity&q=" + question + "&answers=1&site=stackoverflow";
```

Comment:
Too long. limit to 80-120 characters per line. Distribute in separate lines accordingly.